packages/in_app_purchase/in_app_purchase_storekit/test/fakes/fake_storekit_platform.dart

```diff
@@ -30,6 +30,7 @@ class FakeStoreKitPlatform {
   PlatformException? restoreException;
   SKError? testRestoredError;
   bool queueIsActive = false;
+  Map<String, dynamic>? discountReceived;
```

Comment:
nits: Alternatively, we can make it non-nullable and initialized it to empty map, this can avoid many unwrapping.

Reply:
Done. Changed in the last commit.